core/mgmt/v1beta/metric.proto

```diff
@@ -155,7 +153,7 @@ message ListPipelineTriggerChartRecordsResponse {
 // represent the total credit consumption in a given time bucket.
 message CreditConsumptionChartRecord {
   // Credit owner ID, e.g. `users/chef-wombat`.
-  string credit_owner = 1 [(google.api.field_behavior) = OUTPUT_ONLY];
+  string owner_id = 1 [(google.api.field_behavior) = OUTPUT_ONLY];
```

Comment:
I'm thinking that we should use `namespace_id` here.

Reply:
Yeah I was hesitant, because a namespace _owns_ credit so credit owner is valid and quite spread. But I think the change won't generate any confusion so I'm fine with it.

In the pipeline trigger endpoints, should I use `namespace_id` as well?